### PR TITLE
11.0 mig analytic tag dimension sale warning

### PR DESCRIPTION
**Pull request by mpanarin**

### CODE REVIEW

**Review by simahawk — APPROVED** [review]

**Comment by simahawk** [comment]
@mpanarin remove `[WIP]` if not needed

File: analytic_tag_dimension_sale_warning/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "name": "Analytic Tag Dimension Sale Warning",
+    "summary": "Group Analytic Entries by Dimensions",
+    "category": "Generic Modules/Accounting",
+    "version": "11.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "Camptocamp SA,Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/account-analytic",
+    "depends": [
+        'analytic_tag_dimension',
+        'sale'
```
Comment (ghost) [inline analytic_tag_dimension_sale_warning/__manifest__.py]: Trailing comma please

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by melroy89 — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Comment by simahawk** [comment]
REMINDER: depends on https://github.com/OCA/account-analytic/pull/146 => update oca_dependencies.txt before merge

File: oca_dependencies.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# list the OCA project dependencies, one per line
+# add a github url if you need a forked version
+account-analytic https://github.com/arkostyuk/account-analytic 11.0-mig-analytic_tag_dimension
```
Comment (grindtildeath) [inline oca_dependencies.txt]: As #146 is merged, you can remove this line.

**Review by grindtildeath — COMMENTED** [review]
oca_dependencies to update, otherwise LGTM ! 👍

**Review by yvaucher — APPROVED** [review]

**Pull request merged**